### DISCUSSION TITLE
Azure Sentinel notifications

### DISCUSSION THREAD

**Discussion by poprostupiotr** [AzureCommunityPL/group#149] [Pomoc]
Cześć, mam małe pytanie odnośnie powiadomień w Azure Sentinel.
Czy ktoś może konfigurował wysyłanie powiadomień po przypisaniu incydentu do danego usera?
Incydent -> przypisanie incydentu do usera -> powiadom usera o przypisaniu do niego incydentu
Będę wdzięczny za wszelkie rady. Z góry dzięki!

**Comment by wi5nia** [comment]
Chyba sie nie da AFAIK, na [User Voice](https://feedback.azure.com/forums/920458-azure-sentinel/suggestions/41828767-add-incident-assignment-notification-alert-to-azur) jest item do tego
